jest.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  env: {
+    jest: true,
+  },
+  extends: ['plugin:jest/recommended', './base.js'],
```

Comment:
I assumed that it would sequentially do the extends, so that doing base last will keep prettier last, but [this guide](http://eslint.org/docs/developer-guide/shareable-configs) doesn't help me further. Do you know how to find out the actual order that's used?